### ISSUE TITLE
DetectNet on grayscale images

### ISSUE THREAD

**Issue by ShervinAr** [NVIDIA/DIGITS#1573]
Hello,

Does anyone have any experience in training DetectNet on grayscale images? Any comments in this regard are very much appreciated.

**Comment by Hypernation1** [comment]
I'm trying to train grayscale images, but things are not good... I couldn't make mAP not zero.

**Comment by ShervinAr** [comment]
@Hypernation1  thanks for your response.
Could you please let me know what changes need to be done to convert the color version of DetectNet to a grayscale network?
Or alternatively, could you please post the files you changed so that I could give it a try?

**Comment by RomanSteinberg** [comment]
I read a lot of threads and there are a lot of people who have 0 mAP for color images. So, I think the problem lies in understanding how to push DetedtNet to start detecting!

**Comment by Hypernation1** [comment]
@ShervinAr  I just treated dataset like it is made by RGB.

**Comment by Hypernation1** [comment]
@RomanSteinberg Yeah I think so. I don't know lot about neural network. So I tried many things in dataset and detectnet code, and finally made mAP not zero.

**Comment by RomanSteinberg** [comment]
@Hypernation1 can you give me some tips about it?

**Comment by ShervinAr** [comment]
@RomanSteinberg  Could you please elaborate more the steps needed to get the DetectNet run on grayscale images?
These may include any steps needed to create a gray scale dataset in Digits as well as the changes done in the network definition prototxt file possibly along with other changes to some codes...?

**Comment by RomanSteinberg** [comment]
@ShervinAr I am sorry, but not. My point is that it is hard to train DetectNet even for colored images. So, if you experience in it then it will be easier for you to switch to grayscale. Because right now you can't figure out if your mAP=0 caused by grayscale or by some other reasons.

**Comment by ShervinAr** [comment]
@RomanSteinberg  For your information, I have already been able to train 5 different deep CNN structures for Cars/Pedestrians and joint Car&pedestrian detection in Digits, all with reasonable mAP of ~ 80. Now I am planning to convert the models to grayscale and then train them in digits and I was wondering whether you might share your experience with us...

**Comment by Hypernation1** [comment]
I had an error in dataset form. Also I edited xsize and ysize to suit on my data.
